docs/api/1.x/synchronisation.rst

```diff
@@ -13,67 +13,129 @@ This section describes the basic aspects of synchronisation using *Kinto*.
-#. If response is ``304 Not modified``, done → up to date, nothing to do.
-#. If response is ``200 OK`` store ``ETag`` response header value for next
-   synchronisation and handle the obtained list of changes.
+Depending on the context (latest first, readonly, etc.), there are several
+strategies to poll the server for changes.
 
-.. note::
+.. important::
 
     * Deleted records have an attribute ``delete: true``.
     * Created/updated records are both returned in their new version.
     * Since *Kinto* does not keep any history, there is no *diff* for updates.
 
 
-Paginated changes
------------------
+Pagination
+----------
 
-By default, *Kinto* does not paginate the records list. If an explicit limit is
-set in the server settings or using the ``?_limit=<nb>`` parameter, then polling for
-changes will be paginated.
+By default, *Kinto* does not paginate the records list. Since an explicit limit can
+be set in the server settings, clients must handle pagination when polling for
+changes.
 
-It basically consists in fetching the list until the ``Next-Page`` is present.
+In order to reduce the size of response payloads, the client can also force the
+pagination using the ``?_limit=<nb>`` querystring parameter.
 
-Since changes can occur between the first and the last page, the synchronisation
-process is a bit more complex.
+Pagination basically consists in fetching the list until the ``Next-Page`` response header
+is present. The ``Next-Page`` header is the **full** URL of the next page.
 
-#. Cold sync → no ``?_since``.
-#. ``GET /buckets/default/collections/articles?_since=<timestamp>`` (with header ``If-Match: <timestamp>``)
-#. If response is ``304 Not modified``, done → up to date, nothing to do.
-#. If no ``Next-Page`` response header, done → store ``ETag`` response header value for next
-   synchronisation.
-#. If ``Next-Page`` response header is present → store the ``ETag`` response header value into a variable ``timestamp``
-   and go on to it (it's an url) using a ``If-Match: <timestamp>`` request header.
-#. If response is ``200 OK`` → repeat previous step.
-#. If response is ``412 Precondition Failed``, some changes occured since the last
-   page → Store the ``ETag``response header into ``before``
-#. Fetch and handle the changes using ``GET collections?_since=<timestamp>&_before=<before>``,
-   and store the ``ETag``response header in to ``timestamp``
-#. Go back to step 5 (follow the ``Next-Page``)
+.. note::
 
+    Pagination requests carry every necessary parameter to be reproduced in case
+    of connectivity error.
 
-Apply changes
-=============
+
+Strategy #1 — Oldest first
+--------------------------
+
+The simplest way to obtain the changes is to sort the records by timestamp
+ascending.
+
+We will use ``sort=last_modified`` and ``_since=<timestamp>``:
+
+#. First sync: ``timestamp := 0``
+#. Next sync: ``timestamp := MAX(local_records['last_modified'])``
```

Comment:
> because it may be that two or more changes happened during the same timestamp

This is not possible on server side. Even if two records happens at the same timestamp, they will have different timestamps.

See http://kinto.readthedocs.org/en/latest/api/1.x/cliquet/timestamps.html?highlight=timestamp